ledger/ledger-on-memory/README.md

```diff
@@ -0,0 +1,43 @@
+# Overview
+This document is to help internal engineers work with the ledger-on-memory.
+Ledger on memory is a kv-based ledger that uses a simple in-memory map as an underlying
+storage, it uses either H2 or Postgres as its index db.
+
+# Ledger On Memory
```

Comment:
Probably worth mentioning you can also run `bazel run //ledger/ledger-on-memory:app`.

Reply:
Done